package.json

```diff
@@ -1,39 +1,145 @@
+    "bootstrap": "^4.3.1",
+    "camelcase": "^5.2.0",
+    "case-sensitive-paths-webpack-plugin": "2.2.0",
+    "copy-to-clipboard": "^3.2.0",
+    "core-js": "^3.6.0",
+    "cross-env": "^6.0.3",
+    "css-loader": "3.0.0",
+    "dotenv": "8.0.0",
+    "dotenv-expand": "5.1.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "eslint": "^5.16.0",
+    "eslint-config-react-app": "^4.0.0",
+    "eslint-loader": "2.1.2",
+    "eslint-plugin-flowtype": "3.10.3",
+    "eslint-plugin-import": "2.17.3",
+    "eslint-plugin-jsx-a11y": "6.2.1",
+    "eslint-plugin-react": "7.13.0",
+    "eslint-plugin-react-hooks": "^2.2.0",
+    "fetch-mock": "^8.2.0-beta.2",
+    "file-loader": "4.0.0",
+    "fs-extra": "8.0.1",
+    "gh-pages": "^2.0.1",
+    "html-webpack-plugin": "4.0.0-beta.5",
+    "identity-obj-proxy": "3.0.0",
+    "is-wsl": "^2.0.0",
+    "jest": "24.8.0",
+    "jest-environment-jsdom-fourteen": "0.1.0",
+    "jest-resolve": "24.8.0",
+    "jest-watch-typeahead": "0.3.1",
+    "jquery": "^3.4.1",
+    "js-file-download": "^0.4.8",
+    "less-loader": "^5.0.0",
+    "less-vars-to-js": "^1.3.0",
+    "mini-css-extract-plugin": "0.7.0",
+    "optimize-css-assets-webpack-plugin": "5.0.1",
+    "pnp-webpack-plugin": "1.5.0",
+    "postcss-flexbugs-fixes": "4.1.0",
+    "postcss-loader": "3.0.0",
+    "postcss-normalize": "8.0.1",
+    "postcss-preset-env": "6.6.0",
+    "postcss-safe-parser": "4.0.1",
+    "react": "^16.12.0",
+    "react-app-polyfill": "^1.0.0",
+    "react-dev-utils": "^10.0.0",
+    "react-docgen-typescript-loader": "^3.6.0",
+    "react-dom": "^16.12.0",
+    "react-ga": "^2.5.3",
+    "react-intl-universal": "^2.0.4",
+    "react-router-dom": "^5.1.2",
+    "react-router-prop-types": "^1.0.4",
+    "react-scripts": "^3.3.0",
+    "react-scroll-parallax": "^2.0.1",
     "react-simple-chatbot": "^0.6.1",
-    "styled-components": "^4.3.2"
-  },
-  "devDependencies": {
-    "typescript": "^3.6.3"
+    "react-slick": "^0.24.0",
+    "resolve": "1.11.0",
+    "sass-loader": "7.1.0",
+    "scriptjs": "^2.5.9",
+    "semver": "6.1.1",
+    "sharethis-reactjs": "^1.3.3",
+    "storybook": "^5.1.11",
+    "style-loader": "0.23.1",
+    "styled-components": "^4.4.1",
+    "terser-webpack-plugin": "1.3.0",
+    "url-loader": "2.0.0",
+    "webpack": "^4.41.4",
+    "webpack-cli": "^3.3.10",
+    "webpack-dev-server": "3.7.1",
+    "webpack-merge": "^4.2.2",
+    "webpack-pwa-manifest": "^4.0.0",
+    "workbox-webpack-plugin": "^4.3.1"
   },
   "scripts": {
-    "start": "yarn build-component-types && cd app && yarn start",
-    "build": "yarn build-components && cd app && yarn build",
-    "build-component-types": "cd components && tsc --EmitDeclarationOnly",
-    "build-components" : "cd components && yarn build",
-    "test": " cd app && yarn test",
-    "storybook": " cd storybook && yarn storybook",
-    "deploy": "cd app && yarn deploy"
+    "start": "BROWSER=none node scripts/start.js",
```

Comment:
While we're here, let's see if we can omit `BROWSER=none` from this command. It's unix only so would break on Windows systems running the app.

Reply:
Has anyone using Windows complained about it? We should advise anyone using Windows to use WSL. It's a built in feature and has been part of a Windows 10 for a few years now. It's a go to for any cross platform development since it has a full bash compatibility.

I personally prefer not to run a browser at startup since it opens a page in my default browser which I'm not using for development.

Reply:
@dusanradovanovic We'll chat over Slack. There was a post that came in.